trino-connector/src/main/java/com/datastrato/gravitino/trino/connector/catalog/CatalogConnectorAdapter.java

```diff
@@ -22,7 +23,10 @@ default List<PropertyMetadata<?>> getTableProperties() {
   }
 
   /** @return Return internal connector config with Trino. */
-  Map<String, Object> buildInternalConnectorConfig(GravitinoCatalog catalog) throws Exception;
+  Map<String, String> buildInternalConnectorConfig(GravitinoCatalog catalog) throws Exception;
```

Comment:
Is this change a result of the interfaced modification introduced by Trino 435?

Reply:
It's modified by a newly created connector mechanism.